desktop/integration-test/websocket.test.ts

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import { spawn, ChildProcess } from "child_process";
+
+import { launchApp } from "./launchApp";
+
+describe("websocket connection", () => {
+  let sysmonProcess: ChildProcess;
+
+  // Start the sysmon process using npx
+  function startWebSocketServer() {
+    sysmonProcess = spawn("npx", ["@foxglove/ws-protocol-examples@latest", "sysmon"]);
+  }
+
+  async function closeWebSocketServer() {
+    return await new Promise((resolve) => {
+      // Close all channels to ensure that there is not any async leak
+      sysmonProcess.stdin?.end();
+      sysmonProcess.stdout?.destroy();
+      sysmonProcess.stderr?.destroy();
+
+      // Kill the process
+      sysmonProcess.kill();
+
+      resolve(1);
+    });
+  }
+
+  it("should show correct attributes using sysmon ws and raw messages panel", async () => {
```

Comment:
It would be nice explaining sysmon websocket to make it clearer for future developers

Reply:
it was changed to our own ws server